lib/modules/manager/argocd/extract.spec.ts

```diff
@@ -7,6 +17,15 @@ const randomManifest = Fixtures.get('randomManifest.yml');
 const validApplicationSet = Fixtures.get('validApplicationSet.yml');
 
 describe('modules/manager/argocd/extract', () => {
+  describe('createDependency', () => {
+    it('return null for kubernetes manifest', () => {
+      const result = createDependency(
+        load(randomManifest) as ApplicationDefinition
+      );
+      expect(result).toEqual([]);
+    });
+  });
+
```

Comment:
```suggestion
```
I have meant  something like this: 
```ts
    it('return null for ArgoCD Project', () => {
      const result = extractPackageFile(
        `
apiVersion: argoproj.io/v1alpha1
kind: AppProject
      `,
        'applications.yml'
      );
      expect(result).toBeNull();
    });
```
that way you do not have to load anything additionally and don't need to export `createDependency`.

Reply:
That particular test case is being handled by [the following test](https://github.com/renovatebot/renovate/pull/20648/files/a50b5e91e25cc21ec6b02bad6df336a73bb7c38a#diff-aa1a1d7e19235589677f194fe5f4183b5f9b2fa8c878bcbc7d3adf358918a816R41). The issue is that the validation [in here](https://github.com/renovatebot/renovate/pull/20648/files/a50b5e91e25cc21ec6b02bad6df336a73bb7c38a#diff-de5a018e5a8529dc329235872d68a919609bbdbb0e15bf7d19821fea28df153aR46) will always return `null` because the input resource is not an argo Application or an ApplicationSet. As such, [this line in here](https://github.com/renovatebot/renovate/pull/20648/files/a50b5e91e25cc21ec6b02bad6df336a73bb7c38a#diff-de5a018e5a8529dc329235872d68a919609bbdbb0e15bf7d19821fea28df153aR31) will not be covered by tests because it is never executed.